### PR TITLE
added isort linter style

### PR DESCRIPTION
**Pull request by Mikhail-Gorelov**
added isort linters style

### CODE REVIEW

File: web/pyproject.toml [hunk 1]
```diff
@@ -60,3 +60,9 @@ ignore_missing_model_attributes = true
 
 [tool.doc8]
 max_line_length = 110
+
+[tool.isort]
+multi_line_output = 3
+skip = "migrations"
+line_length = 88
+include_trailing_comma = true
```
Comment (bandirom) [inline web/pyproject.toml:68]: lets add profile settings which will be compatible with black. Line lenth 110

Comment (Mikhail-Gorelov) [reply web/pyproject.toml:68]: Got it

**Comment by bandirom** [comment]
Lets also add Isort to CICD Github actions and Gitlab :)

File: web/pyproject.toml (hunk 1 above)
Comment (Mikhail-Gorelov) [reply web/pyproject.toml]: resolved this issue in c9ec021d3d35b28b254470662efc97b210c7035f commit

**Comment by bandirom** [comment]
and need to add to local.txt this lib

**Comment by Mikhail-Gorelov** [comment]
> and need to add to local.txt this lib

added to lib in 3f5f1f43f39da257bec3a4d1a658fd3ae1d2c8e5 commit

**Comment by Mikhail-Gorelov** [comment]
> Lets also add Isort to CICD Github actions and Gitlab :)

Added in commit 662bf6d7bc41beae0ec66b1f677940df4bf95eb4

**Comment by Mikhail-Gorelov** [comment]
resolved all current issues

**Pull request merged**